cmd/gpu_plugin/fakedev/intel-gpu-plugin-fake.yaml

```diff
@@ -0,0 +1,108 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: intel-gpu-plugin
+  namespace: inteldeviceplugins-system
+  labels:
+    app: intel-gpu-plugin
+spec:
+  selector:
+    matchLabels:
+      app: intel-gpu-plugin
+  template:
+    metadata:
+      labels:
+        app: intel-gpu-plugin
+    spec:
+      serviceAccountName: resource-reader-sa
+      initContainers:
+      - name: fakedev-generator
+        image: intel/fakedev-generator:latest
+        securityContext:
+          runAsUser: 0
+          readOnlyRootFilesystem: false
+          allowPrivilegeEscalation: false
+        # files are generated under CWD
+        workingDir: /tmp/fakedev
+        volumeMounts:
+        - name: devfs
+          mountPath: /tmp/fakedev/dev
+          readOnly: false
+        - name: sysfs
+          mountPath: /tmp/fakedev/sys
+          readOnly: false
+        - name: fake-conf
+          mountPath: /config
+          readOnly: true
+        # generate fake sysfs / devfs files for GPU plugin based on config
+        command: ["/generator", "-json", "/config/fakedev.json", "-verbose"]
+      - name: intel-gpu-initcontainer
+        image: intel/intel-gpu-initcontainer:0.24.0
+        imagePullPolicy: IfNotPresent
+        securityContext:
+          readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
+        volumeMounts:
+        - name: sysfs
+          mountPath: /host-sys
+          readOnly: true
+        - name: nfd-features
+          mountPath: /nfd
+          readOnly: false
+        workingDir: /usr/local/bin/gpu-sw
+        # convert generated sysfs content to NFD feature labels
+        command: ["sh", "-c", "./intel-gpu-nfdhook | tee /nfd/fake-gpu"]
+      containers:
+      - name: intel-gpu-plugin
+        env:
+          - name: NODE_NAME
+            valueFrom:
+              fieldRef:
+                fieldPath: spec.nodeName
+        image: intel/intel-gpu-plugin:latest
+        securityContext:
+          readOnlyRootFilesystem: true
+          allowPrivilegeEscalation: false
+        command: [
+          "/usr/local/bin/intel_gpu_device_plugin",
+          "-shared-dev-num", "2",
+          "-fake-mode", "/tmp/fakedev",
```

Comment:
This should probably be "--prefix" ?

Reply:
Yes, apparently I messed up my Friday evening rebase.  Will fix it again.  Thanks!